### PR TITLE
emtpy and null tag values for series raises error

### PR DESCRIPTION
**Pull request by KutuzovGregory**

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/method/series/SeriesInsertTest.java [hunk 1]
```diff
@@ -539,4 +603,74 @@ public void testMillisecondsUnsupported() throws Exception {
...
+        Series series = new Series("e-empty-tag-1", "m-empty-tag-1");
+        series.addData(new Sample(1465502400000l, "1"));
+
+        series.addTag("empty-tag", "");
+
+        Response response = insertSeries(series);
+        String errorMessage = response.readEntity(String.class);
+
+        assertEquals("Incorrect response status code", BAD_REQUEST.getStatusCode(), response.getStatus());
+        assertEquals("{\"error\":\"IllegalArgumentException: Tag \\\"empty-tag\\\" has empty value\"}", errorMessage);
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/series/SeriesInsertTest.java]: What about moving out the error message to class constant?
`EMPTY_TAG_ERROR_MESSAGE` as example

Comment (KutuzovGregory) [inline src/test/java/com/axibase/tsd/api/method/series/SeriesInsertTest.java]: This constant includes particular tag name.

Comment (unrealwork) [inline src/test/java/com/axibase/tsd/api/method/series/SeriesInsertTest.java]: Create a `String.format` template constant message.

Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/series/SeriesInsertTest.java]: Then what about regexp pattern?

**Pull request merged**